Modules/Core/Common/include/itkCommonEnums.h

```diff
@@ -212,7 +212,10 @@ class OctreeEnums
   enum class Octree : uint8_t
   {
     UNKNOWN_PLANE,   ///< The plane is Unknown
-    SAGITAL_PLANE,   ///< The plane is Sagittal
+    SAGITTAL_PLANE,  ///< The plane is Sagittal
+#if !defined(ITK_LEGACY_REMOVE)
+    SAGITAL_PLANE = SAGITTAL_PLANE, ///< Support misspelling
```

Comment:
There is a [[deprecated]] attribute that can be used on enums, and may be appropriate here.

Reply:
I added it.